main/exercice/overview.php

```diff
@@ -155,7 +155,7 @@
     $btn_class = '';
 }
 
-if (!empty($attempts) && $visible_return['value'] == true) {
+if (!empty($attempts)) {
```

Comment:
OK, but I expect an explanation about the removal of $visible_return['value'] == true...
If it was removed, then what was it used for before that? (something that avoids the necessity for the reader to review all the code to get to the same conclusion)